argparse_dataclass.py

```diff
@@ -206,12 +206,15 @@
 
 if sys.version_info[1] >= 8:
     # get_args was added in Python 3.8
-    from typing import get_args
+    from typing import get_args, get_origin
```

Comment:
`get_origin` is only available on Python 3.8+. I think I'm fine with that (it's really annoying to find out if `Optional` was there without using this) but then `setup.py` needs some updates to reflect this (and likely the tests on CI as well).

Reply:
Ah, I completely missed that! Thanks!